.travis.yml

```diff
@@ -3,9 +3,9 @@ sudo: false
 language: java
 
 jdk:
-  - openjdk6
+#  - openjdk6
 #  - openjdk7
-#  - oraclejdk7
+   - oraclejdk7
```

Comment:
I have stumbled over this some time ago also in another (related) project: https://github.com/uwolfer/gerrit-intellij-plugin/commit/0eaa71aa8c3c952944be0efb571d24bfeb21a5d6

JDK 1.6 does not accept the plus sign (+). In your case, it cannot be fixed as easy as I have done in the example above since Gson does the parsing stuff.

I see two solutions (since I would like to keep JDK 6 compatibility):
1) Workaround: Change the value type of approvals to String in order to prevent number parsing.
2) Write an own Gson TypeAdapter which handles numbers with plus like in the example above. Probably a starting point: http://stackoverflow.com/a/24945809 (there are other TypeAdapters around in this project).

Feel free to decide how you would like to do it. Otherwise, the change is ready to get merged IMHO :)